### PR TITLE
Add "need admin approval" question to FAQ

### PR DESCRIPTION
**Pull request by nikmd23**

### CODE REVIEW

File: vs-online/resources/faq.md [hunk 1]
```diff
@@ -56,6 +56,10 @@ Visual Studio Codespaces will clone and utilize the user-provided source and/or
 
 Visual Studio Codespaces is eligible under the Microsoft Azure Bounty Program. For information, visit <https://www.microsoft.com/msrc/bounty-microsoft-azure>.
 
+### How do I register the proper Codespaces Azure resource provider if I receive the "needs admin approval" message?
```
Comment (jkeech) [inline vs-online/resources/faq.md:59]: I think this is confusing two different issues. The admin consent request is for the AAD app, which is not related to the RP registration. Users need to get the AAD tenant admin(s) to consent to using our application in their tenant.

Comment (nikmd23) [reply vs-online/resources/faq.md:59]: oh, thanks for the info @jkeech. What error message is surfaced when the user needs to register the RP?

Also, is there a link for the AAD tenant admin/consent that we can use instead here? My GoogleFoo is failing me.

Comment (legomushroom) [reply vs-online/resources/faq.md:59]: The admin should use these URLs to consent:

- https://login.microsoftonline.com/{tenant}/adminconsent?client_id=a3037261-2c94-4a2e-b53f-090f6cdd712a&redirect_uri=https://online.visualstudio.com
- https://login.microsoftonline.com/{tenant}/adminconsent?client_id=9db1d849-f699-4cfb-8160-64bed3335c72&redirect_uri=https://online.visualstudio.com

Comment (nikmd23) [reply vs-online/resources/faq.md]: Alright, thanks @legomushroom and @jkeech. I've updated the text and added in the links.

**Pull request merged**